worlds/zillion/options.py

```diff
@@ -41,6 +41,12 @@ class VBLR(Choice):
     option_restrictive = 3
     default = 1
 
+    def to_zz_vblr(self) -> ZzVBLR:
```

Comment:
what's the point of this? is there a bug in the options?

Reply:
no bug
It's replacing a `typing.cast` that was in the `validate` function below.
As I was reworking that function, I decided this was a better place to handle it.
The API of `zilliandomizer` takes a `typing.Literal`, while `Option.current_key` gives a `str`.

Reply:
and mypy doesn't like an override that returns a different type than its parent?

Reply:
It does with some categories of expressions. https://mypy.readthedocs.io/en/stable/type_narrowing.html
What is not on that list is the `in` operator (used in the line `assert key in`...).
Nothing in that type narrowing list works for literals.
Here is one of the mypy issues: https://github.com/python/mypy/issues/12535

Reply:
After I shared that link with you, I found a way that mypy can narrow to `Literal`.
I just had to scroll down to https://mypy.readthedocs.io/en/stable/type_narrowing.html#user-defined-type-guards